#ifndef ROS_NODE_ROSENBROCK_H
#define ROS_NODE_ROSENBROCK_H

#define ROS_NODE_ROSENBROCK_NODE_NAME "open_node"
#define ROS_NODE_ROSENBROCK_SOLUTION_TOPIC "result"
#define ROS_NODE_ROSENBROCK_PARAMS_TOPIC "parameters"
#define ROS_NODE_ROSENBROCK_RATE 35
#define ROS_NODE_ROSENBROCK_SOLUTION_TOPIC_QUEUE_SIZE 100
#define ROS_NODE_ROSENBROCK_PARAMS_TOPIC_QUEUE_SIZE 100
#define ROS_NODE_ROSENBROCK_DEFAULT_INITIAL_PENALTY 890.0

#endif /* Header Sentinel: MPC_DUMMY_H */
